python/secure_install/pipeline_scripts/create_install_archive.bash

```diff
@@ -18,7 +18,7 @@ cp README.md $install_dir
 cd $install_dir
 
 pip wheel ../../
-pip-compile --generate-hashes ../../setup.py
+pip-compile --generate-hashes pyproject.toml
```

Comment:
Still located same place, so shouldn't drop the drill down for the file path, I don't believe.

Reply:
Unrelated, and definitely confusing me, but seems we've missed changing this in the other repos, so not sure how it's gone through fine in pipeline....Wooo